src/api/projectApi.ts

```diff
@@ -0,0 +1,29 @@
+import Axios from 'axios';
+import { Dispatch } from 'redux';
+
+import { addNotification } from '../redux/actions/notificationActions';
+import i18n from '../translations/i18n';
+import { StatusCode } from '../utils/constants';
+import { getIdToken } from './userApi';
+
+export enum ProjectsApiPaths {
+  PROJECTS = '/cristin-projects',
+}
+
+export const searchProjects = async (query: string, dispatch: Dispatch) => {
+  try {
+    const idToken = await getIdToken();
+    const response = await Axios.get(`${ProjectsApiPaths.PROJECTS}?${query}`, {
+      headers: {
+        Authorization: `Bearer ${idToken}`,
+      },
+    });
+    if (response.status === StatusCode.OK && response.headers) {
```

Comment:
hvorfor sjekkes `response.headers` her? Tror den er `undefined` for mocking (?)

Reply:
bra spottet. Den var blitt med fra et cristin-kall der vi fikk antall treff i header.